Keas.Mvc/Services/SecurityService.cs

```diff
@@ -108,7 +113,29 @@ private bool IsInRole(Role role, Team team)
             var user = await _dbContext.Users.SingleOrDefaultAsync(x => x.Email == userId);
             return user;
         }
-        
+
+        public async Task<List<User>> GetUsersInRoles(List<Role> roles, int teamId)
```

Comment:
@jcubbage this doesn't seem to do what it says -- I don't see where you are finding the users in the given list of roles.  Maybe you need a .contains in the where statement?

Reply:
Dropped that when I refactored it. Was originally:
using (_dbContext) 
            { 
                var team = await _dbContext.Teams.SingleAsync(t => t.Id == teamId); 
 
                _dbContext.Entry(team) 
                    .Collection(t => t.TeamPermissions) 
                    .Query() 
                    .Where(tp => roles.Contains(tp.Role)) 
                    .Include(tp => tp.User) 
                    .Load(); 
....}